app-frontend/src/components/App/index.js

```diff
@@ -0,0 +1 @@
+// export { default } from './App';
```

Comment:
barrel file with just a comment?
You shouldn't commit commented out code at all